packages/app-builder-lib/src/util/NodeModuleCopyHelper.ts

```diff
@@ -130,17 +133,29 @@ export class NodeModuleCopyHelper extends FileCopyHelper {
         CONCURRENCY
       )
 
+      let isEmpty = true
       for (const child of sortedFilePaths) {
         if (child != null) {
           result.push(child)
+          this.metadata.get(child)?.isSymbolicLink() && symlinkFiles.set(child, result.length - 1)
```

Comment:
To maintain code consistency, let's continue using bracketed if-statements.
```
if (this.metadata.get(child)?.isSymbolicLink()) {
   symlinkFiles.set(child, result.length - 1)
}
```

Reply:
Fixed.